README.md

```diff
@@ -769,7 +769,39 @@ Migrating from Folktale should be *very* straightforward: many of the names are
 
 In many cases, you can simple rename your imports and some of the function invocations for Folktale to switch to True Myth – several imports are supplied with exactly that pattern in mind. If a given item is not mentioned, you can assume no change other than the import is required.
 
-- [ ] TODO: migration path from Folktale 1.0
+##### Update imports
+
+```diff
+-import Maybe from 'data.maybe'
++import { Maybe } from 'true-myth'
+```
+
+##### Update `Nothing`/`Just` constructors
+```diff
+-Maybe.Nothing()
+-Maybe.Just('foo')
++Maybe.nothing()
++Maybe.just('foo)
+```
+
+##### Update `getOrElse` -> `unwrapOr`
+```diff
+-foo.getOrElse('oh noes')
```

Comment:
You might add a note here that `getOr` is also available as an alias.

Reply:
So interestingly, `getOrElse` is supposed to be an alias as well, but it didn't work. Rather than fixing these, I vote that we just take these aliases out because it's equally as easy to `s/getOrElse/getOr` vs. `s/getOrElse/unwrapOr`.

Reply:
Hmm. `getOr` is the replacement here, I think? `getOrElse` is an alias for `unwrapOrElse`, but the only reason it's there is for compatibility. (That said, I don't want to make a breaking change already!)

Reply:
Thinking more on this: having `getOrElse` is actually probably more confusing that not, given that `getOrElse` has *very* different semantics. Blast.